Apps/Immunization/src/Startup.cs

```diff
@@ -64,9 +65,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IVaccineStatusService, VaccineStatusService>();
 
             // Add delegates
-            services.AddTransient<IImmunizationDelegate, RestImmunizationDelegate>();
+            services.AddTransient<Delegates.IImmunizationDelegate, Delegates.RestImmunizationDelegate>();
```

Comment:
Nit: it doesn't look like these names need to be qualified with the using statement that was added.

Reply:
theres a delegate with the exact  same name in common